src/main/webapp/app/course/manage/course-management-exercises.component.html

```diff
@@ -1,33 +1,100 @@
 <ng-container *ngIf="course">
-    <h3>
-        <span
-            >{{ course.title }} - {{ quizExercisesCount + programmingExercisesCount + modelingExercisesCount + fileUploadExercisesCount + textExercisesCount }}
-            {{ 'artemisApp.course.exercises' | artemisTranslate }}</span
-        >
-    </h3>
+    <div class="row">
+        <div class="col">
+            <h3>
+                <span>
+                    {{ course.title }}
+                    - {{ getExerciseCount() }}
```

Comment:
Please don't use function calls here but rather an attribute (Coding Guidelines)

Reply:
You should be aware that replacing all function calls with attributes will make the course-management-exercises.component.ts file almost twice as large. The setters for the count attributes alone would take up 40 lines of code.
For this reason, I personally prefer how it is currently, but since the style guide prefers it the other way I will change it.

Reply:
As far as I know Angular has a better performance with attributes since it has to update on change. With an attribute, you just have to listen for a change but Angular will probably have to constantly reevaluate the function to detect changes.

Reply:
I am aware of this, which is why I changed it.
It's just that it is not as concise now.